experiments/jacobian-lens-0.5b/fit_lens.py

```
@@ -0,0 +1,113 @@
+    model = jlens.from_hf(hf, tok)
+
+    import os
+    os.makedirs(OUT, exist_ok=True)
+    lens = jlens.fit(
+        model,
+        prompts=prompts,
+        checkpoint_path=f"{OUT}/fit-checkpoint.pt",
+    )
+    lens.save(f"{OUT}/jacobian_lens_qwen2.5-0.5b-instruct-bf16.pt")
+
+    jlens_rev = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)